kroxylicious-kms-provider-fortanix-dsm/src/main/java/io/kroxylicious/kms/provider/fortanix/dsm/model/SecurityObjectDescriptor.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kms.provider.fortanix.dsm.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Fortanix key descriptor looking up by kid, name or transient key id.
+ *
+ *
+ * @param kid kid
+ * @param name key name
+ * @param transientKey transient key name
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public record SecurityObjectDescriptor(
+                                       @JsonProperty("kid") String kid,
+                                       @JsonProperty("name") String name,
+                                       @JsonProperty("transient_key") String transientKey) {
```

Comment:
I think we should override `toString()` to mitigate the risk that its output gets logged somewhere thus revealing the `transientKey`.

Reply:
transientKey is just an identifier, it is not the key material itself.